### PR TITLE
Feature/qa 454 ban all dhcp clear one

### PR DESCRIPTION
**Pull request by rpromyshlennikov**

### CODE REVIEW

File: mos_tests/environment/fuel_client.py [hunk 1]
```diff
@@ -48,9 +48,10 @@ def get_ssh_to_node(self, ip):
             private_keys=self.admin_ssh_keys
         )
 
-    def get_ssh_to_cirros(self, ip, private_keys):
+    def get_ssh_to_cirros(self, ip, private_keys=None,
+                          creds=('cirros', 'cubswin:)')):
         return SSHClient(
-            host=ip, username="cirros", password=None,
+            host=ip, username=creds[0], password=creds[1],
```
Comment (TimurNurlygayanov) [inline mos_tests/environment/fuel_client.py]: let's use dict instead of tuple here

File: mos_tests/neutron/python_tests/base.py [hunk 2]
```diff
@@ -155,13 +156,11 @@ def check_vm_connectivity(self, timeout=3 * 60):
             self.check_ping_from_vm(server1, self.instance_keypair,
                                     ips_to_ping, timeout=timeout)
 
-    def check_vm_is_connectable(self, vm):
-        """Check that instance is pingable from hypervisor.
+    def check_vm_is_connectible(self, vm, pkeys=None):
```
Comment (TimurNurlygayanov) [inline mos_tests/neutron/python_tests/base.py]: VM availability. "is_connectible" sounds strange :)

File: mos_tests/neutron/python_tests/test_ban_dhcp_agent.py [hunk 3]
```diff
@@ -46,42 +49,89 @@ def create_cirros_instance_with_ssh(self, name='server01',
...
+                    node_to_clear))
+
+        logger.info("Clear DHCP agent on node {0}".format(node_to_clear))
 
-        # Wait to migrate dhcp agent on new controller
-        if wait_for_migrate:
-            err_msg = "DHCP agent wasn't banned, it is still on {0}"
+        # Wait to reschedule dhcp agent
+        if wait_for_rescheduling:
+            err_msg = "Wait for DHCP agent ({0}) rescheduling after clear"
```
Comment (TimurNurlygayanov) [inline mos_tests/neutron/python_tests/test_ban_dhcp_agent.py]: "after clear" sounds strange. Let's describe it in details.

File: mos_tests/environment/fuel_client.py (hunk 1 above)
Comment (rpromyshlennikov) [inline mos_tests/environment/fuel_client.py]: Can I do it like that? 
def get_ssh_to_cirros(self, ip, username='cirros', password='cubswin:)',
                                   private_keys=None):

**Pull request merged**